package.json

```diff
@@ -20,7 +20,7 @@
     "start": "node -r dotenv/config index.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "tdd": "yarn run test --watch --verbose false",
-    "test": "NODE_ENV=test jest --config=jest.config.js"
+    "test": "NODE_ENV=test jest --coverage --config=jest.config.js"
```

Comment:
Generating coverage is a lot slower than just running the tests. I would recommend we create a separate script here just to run code coverage, and not run it every time unit tests are run.